addons/point_of_sale/static/src/css/pos.css

```diff
@@ -2208,6 +2207,8 @@ td {
     text-align: left;
     max-width: 500px;
     -webkit-transform: translate3d(0,0,0);
+    height: fit-content;
```

Comment:
couldn't you put "height:100%" ? fit-content is suppose to be function

Reply:
https://drive.google.com/file/d/1TzQNlGIglwIlDmJG9IcJhbYvPpNJmPt5/view
First part is with height: 100%;
The second part is with height: fit-content;
Not really sure why it's result.

Reply:
I should put min-content or max-content. Both works and they are not css functions.